doc/script_commands.txt

```diff
@@ -6957,6 +6957,34 @@ Examples:
 
 ---------------------------------------
 
+*unitiswalking(<GID>)
+
+This command checks, if a unit is walking or not.
+Returns 1 if the unit is walking, 0 if the unit is not walking and -1 on error.
+
+Note: There's no differentiation between script and client initiated walking.
+
+Example:
+
+prontera,155,185,5	script	Walky	1_F_MARIA,{
+	// Get the character's GID.
+	.@GID = getcharid(CHAR_ID_ACCOUNT);
+
+	// Make the character walk.
+	unitwalk(.@GID, 155, 175);
+
+	// Pausing the script while the character is walking.
+	while (unitiswalking(.@GID) == 1)
+		sleep2(50);
```

Comment:
sleep is bad idea. better use npc timer

Reply:
Using `*npctimer()` commands would cause an infinity loop error and `freeloop()` freezes the script.
So `sleep2()` is the only working solution here.